token/tokens.go

```diff
@@ -0,0 +1,148 @@
+package token
```

Comment:
Perhaps we could split this into **AccessToken**, **RefreshToken**, etc?
To keep things short and sweet. :-)

Reply:
This is a library interface impl., I'd leave it as-is.